db/data.js

```diff
@@ -170,9 +163,9 @@ const produce = [
   {
     foodName: "Bramley Apple",
     imageURL:
-      "https://res.cloudinary.com/dvpop7e9w/image/upload/v1659637899/Seasonal/057-organic-food_pikcaf.png",
+      "https://res.cloudinary.com/dvpop7e9w/image/upload/v1666175319/Seasonal/003-apple_yzjgbk.png",
     description:
-      "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed feugiat euismod augue, ac fringilla diam pharetra id. In porttitor convallis nisi vel finibus. Pellentesque at diam venenatis, ornare mi eget, vestibulum arcu. Vivamus consectetur, erat eu semper dapibus, dolor sem suscipit mi, ut dignissim enim nibh vel risus. Sed aliquet, sapien eu congue gravida, lorem metus bibendum mauris, quis maximus nibh erat eu erat. Cras vulputate purus magna, tincidunt porttitor massa vestibulum sed. Nam turpis massa, dictum eu accumsan in, posuere sit amet lorem. Phasellus est sapien, faucibus eget elit et, blandit consequat ex. Proin feugiat arcu a magna iaculis, nec pharetra orci porta. Proin vel orci ante. Aliquam viverra non ligula nec rhoncus. Ut et tincidunt elit. Phasellus non risus in mi vulputate venenatis vitae quis enim. Vestibulum quis dolor venenatis, lobortis diam in, faucibus turpis. Nullam quis pharetra nulla, vestibulum tincidunt ipsum. Donec non ultrices ligula. Quisque at interdum velit. Donec laoreet elit enim, a vulputate risus cursus eu. Duis et est in mi pulvinar rhoncus in ut lorem. Integer nec dolor et velit sollicitudin vehicula. Duis sapien justo, condimentum nec orci eu, dictum dapibus nisl. Praesent ultricies libero vel leo pulvinar molestie.",
+      "Bramley apples are a large, green cooking apple, which are available year round, but are at their height from December through until March. They can be eaten uncooked, but their taste will be very different, so this is not advisable. Like most apples, bramley apples are high in fibre, vitamins and antioxidants. Make sure to select bramley apples which are unblemished and firm. They can be boiled or stewed in a variety of recipes to make sauces and desserts. An alternative in recipes could be other varieties of apples, or pears. Bramley apple trees are naturally quite resistant to disease, very productive, and fairly easy to grow, suiting the British climate. Apples are one of the easiest fruit trees to grow, and tend to be popular with a lot of home gardeners because of this.",
```

Comment:
Why does Bramley Apple get its own entry separate to Apples?

Reply:
Mostly because I pulled the original list of foods from BBC Good Foods, which lists them separately. Bramley apples are a main cooking apple rather than an eating apple, but I will also say that the icon for Bramley Apple doesn't match: it's not the right colour. So we could delete if you think it's duplicated?